src/integrationtest/python/DocTraversals_tests.py

```diff
@@ -0,0 +1,47 @@
+# Name: Debasish Kanhar
+
+import unittest
+from subprocess import Popen, PIPE
+from JanuaGraphContainer import JanusGraphContainer
```

Comment:
pretty sure `JanuaGraphContainer` should be `JanusGraphContainer`, have you tried to run these tests locally?

Separately we need to figure out why Travis isn't picking up the initial-pr branch. I can't see anything from the travis side so I'm hoping someone with more access can check the repo settings to make sure the integration looks ok.